sympy/stats/joint_rv.py

```diff
@@ -169,7 +165,7 @@ def pdf(self, *args):
     def cdf(self, other):
         assert isinstance(other, dict)
```

Comment:
It would be good if an exception could be raised with an informative message about `other` if this is not the case.
Can you add test(s) for `cdf`?